packages/cli/src/commands/pin/add.ts

```diff
@@ -0,0 +1,31 @@
+import { StreamID } from '@ceramicnetwork/streamid'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+
+export default class Add extends Command<
+  CommandFlags,
+  {
+    streamId: string
+  }
+> {
+  static description = 'Pin Stream'
+  static args = [
+    {
+      name: 'streamId',
+      required: true,
+      description: 'StreamID to be pinned',
+    },
+  ]
+
+  async run(): Promise<void> {
+    this.spinner.start(`Pinning stream ${this.args.streamId}...`)
+    try {
+      const result = await this.ceramic.pin.add(StreamID.fromString(this.args.streamId))
```

Comment:
Is `StreamID.fromString()` required here? Doesn't `pin.add()` accept a string already?

Reply:
This is required by `pin.add()` and all other pin methods. Without it they throw: `Argument of type 'string' is not assignable to parameter of type 'StreamID'`

Reply:
OK thanks for the info, I opened an issue for it in https://github.com/ceramicnetwork/js-ceramic/issues/1871